dev/release/post-03-website.sh

```diff
@@ -57,7 +57,7 @@ else
 fi
 
 export TZ=UTC
-release_date=$(LANG=C date "+%-d %B %Y")
+release_date=$(LANG=C LC_TIME=C date "+%-d %B %Y")
```

Comment:
Can we remove `LANG=C`?

```suggestion
release_date=$(LC_TIME=C date "+%-d %B %Y")
```